src/selectors/nodes.test.js

```diff
@@ -352,4 +356,18 @@ describe('Selectors', () => {
       });
     });
   });
+  describe('getNodesWithInputParams', () => {
+    const newMockState = reducer(
+      mockState.animals,
+      toggleTypeDisabled('parameters', true)
+    );
+    const nodeswithInputParams = getNodesWithInputParams(newMockState);
```

Comment:
Think this needs updating to `nodesWithInputParams`? Also I think might be typo below `onject`